bench/bm_case1.cpp

```diff
@@ -668,78 +686,6 @@ inline const boost::ut::suite _simd_tests = [] {
             expect(eq(test::n_samples_consumed, N_SAMPLES)) << "did not consume enough input samples";
         };
     }
-
```

Comment:
Shall we completely drop this test case and/or move this to a expert-level bm_case1 benchmark?

Reply:
What test case do you mean? It's not clear from the code context given by the comment.